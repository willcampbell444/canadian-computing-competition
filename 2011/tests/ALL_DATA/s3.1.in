8
4 508 369
1 1 2
2 11 13
3 44 9
3 49 10
2 10 13
2 12 15
4 136 578
