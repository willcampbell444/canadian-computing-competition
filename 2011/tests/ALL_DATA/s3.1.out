empty
empty
empty
crystal
crystal
empty
empty
empty
